enterprise/utils.py

```diff
@@ -389,8 +387,11 @@ def get_enterprise_customer_for_user(auth_user):
 
     Returns:
         (EnterpriseCustomer): enterprise customer associated with the current user.
+
     """
-    enterprise_customer_user = enterprise.models.EnterpriseCustomerUser.objects.filter(user_id=auth_user.id).first()
+    # pylint: disable=invalid-name
+    EnterpriseCustomerUser = apps.get_model(app_label='enterprise', model_name='EnterpriseCustomerUser')
```

Comment:
@douglashall, this is the canonical Django way to get a model that couldn't be loaded via an import. It's not substantially _prettier_ than including the entire namespace, but it's definitely a better way to handle the situation and should be more flexible later on.